metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -489,6 +489,7 @@ def _init_schema_resolver(self) -> SchemaResolver:
                     platform=self.platform,
                     platform_instance=self.config.platform_instance,
                     env=self.config.env,
+                    batch_size=self.config.schema_resolution_batch_size,
```

Comment:
a number of other sources use `initialize_schema_resolver_from_datahub` as well - does it make sense to tweak those too, or this still pretty experimental?

Reply:
unfortunatelly in my tests it didn't improve the overall performance too much